doc/install/ubuntu-16.04.md

```diff
@@ -1,6 +1,6 @@
 # Installing cjdns on Ubuntu 16.04
 
-This is a short guide how to setup an Ubuntu cjdns box.
+This is a short guide how to setup an Ubuntu cjdns box. Make sure you are running all this command as root either by "sudo su -" or log in as root.
```

Comment:
Can you add "sudo" to the commands which need it rather than encouraging people to do everything as root ?
When they come back to the repo later they won't understand why they cannot git pull...